### PR TITLE
[all] Rename ZEIT Now to Vercel

### PR DESCRIPTION
**Pull request by styfle**
https://vercel.com/blog/zeit-is-now-vercel

### CODE REVIEW

**Review by leo — CHANGES_REQUESTED** [review]
This is not working: https://assets.vercel.com/image/upload/v1581518533/repositories/now-cli/v4.png

File: README.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
-![now](https://assets.zeit.co/image/upload/v1581518533/repositories/now-cli/v4.png)
+![now](https://assets.vercel.com/image/upload/v1581518533/repositories/now-cli/v4.png)
```
Comment (TooTallNate) [inline README.md]: This is a 404

File: packages/now-cli/README.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
-![now](https://assets.zeit.co/image/upload/v1581518533/repositories/now-cli/v4.png)
+![now](https://assets.vercel.com/image/upload/v1581518533/repositories/now-cli/v4.png)
```
Comment (TooTallNate) [inline packages/now-cli/README.md]: This is 404

File: packages/now-cli/src/commands/deploy/legacy.ts [hunk 3]
```diff
@@ -259,7 +259,7 @@ export default async function main(
   quiet = !isTTY;
   ({ log, error, note, debug, warn } = output);
 
-  const infoUrl = 'https://zeit.co/guides/migrate-to-zeit-now';
+  const infoUrl = 'https://vercel.com/guides/migrate-to-zeit-now';
```
Comment (TooTallNate) [inline packages/now-cli/src/commands/deploy/legacy.ts]:
```suggestion
  const infoUrl = 'https://vercel.com/guides/migrate-to-vercel';
```

File: packages/now-cli/src/commands/dns/index.ts [hunk 4]
```diff
@@ -64,7 +64,7 @@ const help = () => {
       ${chalk.cyan(
         `$ now dns add <DOMAIN> <NAME> CAA '<FLAGS> <TAG> "<VALUE>"'`
       )}
-      ${chalk.cyan(`$ now dns add zeit.rocks '@' CAA '0 issue "zeit.co"'`)}
+      ${chalk.cyan(`$ now dns add zeit.rocks '@' CAA '0 issue "vercel.com"'`)}
```
Comment (TooTallNate) [inline packages/now-cli/src/commands/dns/index.ts]:
```suggestion
      ${chalk.cyan(`$ now dns add zeit.rocks '@' CAA '0 issue "example.com"'`)}
```

**Review by TooTallNate — CHANGES_REQUESTED** [review]
Left a couple suggestions

File: README.md (hunk 1 above)
Comment (styfle) [reply README.md]: I reverted to assets.zeit.co

File: packages/now-cli/README.md (hunk 2 above)
Comment (styfle) [reply packages/now-cli/README.md]: I reverted to assets.zeit.co

File: packages/now-cli/src/commands/deploy/legacy.ts (hunk 3 above)
Comment (styfle) [reply packages/now-cli/src/commands/deploy/legacy.ts]: Fixed this and all other links to guides and docs

File: packages/now-cli/src/commands/dns/index.ts (hunk 4 above)
Comment (styfle) [reply packages/now-cli/src/commands/dns/index.ts]: Good idea, fixed 🔧

**Review by TooTallNate — APPROVED** [review]

**Comment by jwdalshlsh987** [comment]
XJwdalshlsh987@gmail.com

**Comment by jwdalshlsh987** [comment]
عربي ممكن

**Pull request merged**